README.md

```diff
@@ -1,131 +1,231 @@
-# Aurae AE
+<h1 align="center">
+  <code>ae</code>
+  <h3 align="center">UNIX inspired CLI Client for Aurae</h3>
+</h1>
 
-[![Test and Format](https://github.com/aurae-runtime/ae/actions/workflows/001-go-ubuntu-latest-make-test-format-lint.yaml/badge.svg?branch=main)](https://github.com/aurae-runtime/ae/actions/workflows/001-go-ubuntu-latest-make-test-format-lint.yaml)
+<div align='center'>
 
-Unix inspired command line client for Aurae written in Go!
+<a href='https://github.com/aurae-runtime/ae/blob/main/go.mod'>
+<img alt="go version" src="https://img.shields.io/github/go-mod/go-version/aurae-runtime/ae?color=grey&logo=go&style=for-the-badge">
+  
+</a>
+  
+<a href="https://github.com/aurae-runtime/ae/blob/main/LICENSE">
+<img alt="license" src="https://img.shields.io/github/license/aurae-runtime/ae?color=grey&logo=apache&style=for-the-badge"/>
+  
+</a>
 
-Contributions and newcomers to the project are welcome. Please see [Getting Involved](https://github.com/aurae-runtime/community#getting-involved) to join the Discord and more.
+</div>
 
-[Signing the CLA](https://cla.aurae.io/) is required for contributions.
+---
 
-## What is ae? Why does it exist?
+> The project welcomes new contributors and developers. Check out the **[Getting Involved](https://github.com/aurae-runtime/community#getting-involved)** section and join the Discord. It is mandatory to sign the **[CLA](https://cla.nivenly.org/)** to contribute.
 
-In order to understand `ae` we should first understand `aer`.
 
-### What is aer?
-We intend to auto generate a command line tool named `aer` built on the Rust client that has an identical scope of a single auraed node.
+<!-- TABLE OF CONTENTS -->
+<h2 id="table-of-contents"> 📑 Table of Contents</h2>
 
-This tool is for "power-users" exists as a way of quickly developing and debugging the APIs as we change them. For example an auraed developer might make a change to an API and need a quick way to test the API locally against a single daemon.
+<details open="open">
+  <summary>Table of Contents</summary>
+  <ol>
+    <li><a href="#about-the-project"> ► About The Project</a></li>
```

Comment:
nit: further down you use this triangle to indicate a "foldable section" but here it's just a link. i think you can drop the triangle.

Reply:
I personally find the triangles very practical and the missing one visually unappealing. You can handle it however you want, just let me know. :)

Reply:
or use an <ul> and lose the symbol? i'm fine with whatever you decide, but i think we shouldn't use the same symbol for two different things.

Reply:
It is fixed in 1ff52cf. I dropped the triangle because it is also used for HTML spoilers. Added a simple point to make a space (visually) between the number and the TOC line.